DiamondLightSource/SynchWeb
# LIMS-1540: Remove references to summary pages/DB

**JIRA ticket**: [TICKET-1540](https://jira.diamond.ac.uk/browse/LIMS-1540)

**Summary**:

Since the summary page was moved to a new and much improved codebase, it would be optimal to remove "dead" bits of code in SynchWeb that could cause confusion.

Improvements to components used elsewhere (such as comboboxes) were retained.

**Changes**:
- Remove references to summary pages/DB in the code

**To test**:
- Go to a DC summary page (https://local-oidc-test.diamond.ac.uk:8082/dc/summary/visit/cm37235-5) and check if the page renders correctly
- Perform basic actions (navigate to a data collection, view/edit containers/proteins, select one of the dashboards in the navbar), ensure they still work correctly
